.github/workflows/test-android-emulator.yaml

```diff
@@ -86,19 +85,49 @@ jobs:
         id: tests_step
         run: |
           TESTS_EXIT_CODE=0
+          # TODO: Change to YAML file if outputs list is fixed
           ew-cli \
-            emulatorwtf.yaml:main \
+            --device model=Pixel2,version=31,gpu=auto \
```

Comment:
Why's that? Why not use the file?

Reply:
There's a bug - only last item from outputs list is pulled when you use a file

Reply:
Okay, let's add a `// FIXME` with a comment explaining that.
And let's also remember to remove this workaround and go back to using `emulatorwtf.yaml` when they fix that problem.